src/main/java/view/OutputFormatter.java

```diff
@@ -30,13 +29,33 @@ public String toLine(Line rawLine) {
         String line = rawLine.getSteps().stream()
                 .map(this::getStep)
                 .collect(Collectors.joining());
-        return "    " + line;
+        return SPACE.repeat(4) + line;
     }
 
     private String getStep(Step step) {
-        if (step.equals(Step.EXIST)) {
-            return BAR + "-----";
+        if (step.isRight()) {
+            return BAR + BRIDGE;
+        }
+        return BAR + SPACE.repeat(5);
+    }
+
+    public String toTargetUnit(Targets targets) {
+        return targets.getTargets().stream()
+                .map(this::getTargetUnit)
+                .collect(Collectors.joining());
+    }
+
+    private String getTargetUnit(Target target) {
+        String name = target.getTarget();
+        if (name.length() < 5) {
+            String leftBlank = SPACE.repeat(4 - name.length());
+            String rightBlank = SPACE;
+            name = leftBlank + name + rightBlank;
```

Comment:
```suggestion
            name = String.format("%4s ", name);
```
정확한 의도는 모르겠지만 String.format을 활용해보세요